drivers/pure/flasharray/volumes.go

```diff
@@ -16,3 +22,29 @@ func (vols *VolumeServices) ListAllAvailableVolumes(params map[string]string, da
 	}
 	return m, nil
 }
+
+func (realms *RealmsServices) ListAllAvailableRealms(params map[string]string, data interface{}) ([]RealmResponse, error) {
```

Comment:
pls donot update realm functions in volume.go , can you pls update realm.go for realm functions and pod.go for pod related functions ?